modules/wri_project/config/install/core.entity_view_display.node.project_detail.full.yml

```diff
@@ -159,7 +160,26 @@ third_party_settings:
                   view_mode: teaser
                   link: false
                 third_party_settings: {  }
-            weight: 8
+            weight: 3
+            additional: {  }
+          -
+            uuid: c096c0b5-cccf-4994-94aa-99a39d69aec7
```

Comment:
Since this is being pulled in on the `main_content` view mode, I don't think we need it here.